db/queries/scores/queries.py

```diff
@@ -0,0 +1,63 @@
+"""Queries which are performed on the `scores` table.
+
+Joins allowed.
+"""
+import json
+from typing import Dict
+from typing import List
+
+from db import db
+from db.models.score import JustScore
+from db.queries.assessment_records._helpers import derive_values_from_json
+from db.schemas import JustScoreMetadata
+from sqlalchemy import func
+from sqlalchemy import select
+from sqlalchemy.orm import defer
+
+
+def get_just_score_for_application_sub_crit(
+    application_id: str, sub_criteria_id: str
+) -> Dict:
+    """get_just_score_for_application_sub_crit executes a query on scores
+    which returns the most recent score for the given application_id and 
+    sub_criteria_id.
+
+    :param application_id: The stringified application UUID.
+    :param sub_criteria_id: The stringified sub_criteria UUID.
+    :return: dictionary.
+    """
+    stmt = select(JustScore).where(
```

Comment:
can stmt be more descriptive?

Reply:
stmt is just the sql query that is used to parse into the search and seems to be best practice to use as in sqlalchemy: https://docs.sqlalchemy.org/en/14/orm/queryguide.html